run.py

```diff
@@ -10,6 +12,10 @@ def create_app(config_filename):
     #bcrypt = Bcrypt(app)
 
 
+    #setup flask-jwt-extended extension
+    app.config['JWT_SECRET_KEY'] = 'secret'
+    jwt = JWTManager(app)
```

Comment:
I see you are linking jwt to the app here but you haven't used this on your user.py file, you are using a newly imported jwt package.